### PR TITLE
May to shall be able to

### PR DESCRIPTION
**Pull request by knoshua**

### CODE REVIEW

File: RPIPs/RPIP-43.md [hunk 1]
```diff
@@ -126,9 +126,8 @@ into shares is defined in [RPIP-46](RPIP-46.md).
 - Newly deposited capital that is awaiting deposit to the beacon chain SHALL be
   excluded from capital distribution, but rather be subject to separate functions
   for `prestake` and `stake` transactions.
-- The megapool MAY be used as the node operator's authorized fee recipient for
-  execution layer rewards if they are not opted into the smoothing pool. These
-  funds shall be subject to rewards distribution.
+- The megapool or the node distributor SHALL be allowed to be used as the node operator's authorized fee recipient for
```
Comment (LongForWisdom) [inline RPIPs/RPIP-43.md:129]: Must it be one of these two in this case? Or are other addresses also valid? If so, should be reworded.

File: RPIPs/RPIP-44.md [hunk 2]
```diff
@@ -54,8 +54,8 @@ The exit functionality for a megapool is specified as follows:
   - This function SHALL either remove the specified validators from the megapool or revert
       - If any of the validators are still active on the beacon chain a successful call of this
         function MUST also exit those validators from the beacon chain
-  - This function MAY be freely called by the Node Operator (as implied by [RPIP-43](RPIP-43.md))
-  - This function MAY be called by any account under the condition: `deficit >= exit_deficit`
+  - The Node Operator SHALL be able  to call this function (as implied by [RPIP-43](RPIP-43.md))
```
Comment (LongForWisdom) [inline RPIPs/RPIP-44.md:57]:
```suggestion
  - The Node Operator SHALL be able to call this function (as implied by [RPIP-43](RPIP-43.md))
```

Comment (LongForWisdom) [reply RPIPs/RPIP-44.md:57]: Remove double-space

File: RPIPs/RPIP-44.md [hunk 3]
```diff
@@ -54,8 +54,8 @@ The exit functionality for a megapool is specified as follows:
...
+  - Anyone SHALL be able to call this function under the condition: `deficit >= exit_deficit`
```
Comment (LongForWisdom) [inline RPIPs/RPIP-44.md:58]:
```suggestion
  - This function MUST be permissionlessly callable under the condition: `deficit >= exit_deficit`
```

Comment (LongForWisdom) [reply RPIPs/RPIP-44.md:58]: Reword. Not sure if better. Is permissionlessly a word?

File: RPIPs/RPIP-46.md [hunk 4]
```diff
@@ -41,15 +41,15 @@ This RPIP is part of a set of proposals motivated by a desire to rework Rocket P
 4. `reth_share` SHALL be defined as `100% - reth_commission`
 5. Distributions of revenue from borrowed ETH MUST respect the defined shares
    1. If shares change between claims, distributions MUST make an effort to account for the different values. For example, a distribution could use a duration-weighted average share. Approximations MAY be used where they significantly reduce complexity and/or costs.
-   2. Legacy minipools are an exception and MAY continue to support earlier distribution methodologies 
-6. `node_operator_commission_share`, `surplus_share`, `increase_no_share_seal_increment`, `increase_no_share_seal_count`, and `allowlisted_controllers` MAY be updated by pDAO vote
-7. `node_operator_commission_share` and `surplus_share`, MAY be updated by an address in the `allowlisted_controllers` array
+   2. Legacy minipools are an exception and SHALL continue to support earlier distribution methodologies 
+6. `node_operator_commission_share`, `surplus_share`, `increase_no_share_seal_increment`, `increase_no_share_seal_count`, and `allowlisted_controllers` SHALL be  be updateable by pDAO vote
```
Comment (LongForWisdom) [inline RPIPs/RPIP-46.md:45]:
```suggestion
6. `node_operator_commission_share`, `surplus_share`, `increase_no_share_seal_increment`, `increase_no_share_seal_count`, and `allowlisted_controllers` SHALL be updateable by pDAO vote
```

File: RPIPs/RPIP-46.md [hunk 5]
```diff
@@ -41,15 +41,15 @@ This RPIP is part of a set of proposals motivated by a desire to rework Rocket P
...
+7. `node_operator_commission_share` and `surplus_share`, SHALL be  be updateable by an address in the `allowlisted_controllers` array
```
Comment (LongForWisdom) [inline RPIPs/RPIP-46.md:46]:
```suggestion
7. `node_operator_commission_share` and `surplus_share`, SHALL be updateable by an address in the `allowlisted_controllers` array
```

File: RPIPs/RPIP-46.md [hunk 6]
```diff
@@ -72,7 +72,7 @@ megapool staked RPL and legacy staked RPL) relative to the total supply of RPL m
 owned by the protocol in its treasury or as a result of `surplus_share` buy backs. The voting
 eligible RPL target is a range between `vote_eligible_target_min` and `vote_eligible_target_max`
 1. Updating `voter_share`:
-   1. A new function SHALL be available to update `voter_share`, which MAY be called by anyone
+   1. A new function SHALL be available to update `voter_share`, which SHALL be callable by anyone
```
Comment (LongForWisdom) [inline RPIPs/RPIP-46.md:75]:
```suggestion
   1. A permissionless function SHALL be available to update `voter_share`
```

Comment (LongForWisdom) [reply RPIPs/RPIP-46.md:75]: Reword. Everything here is new so don't need the qualifier?

File: RPIPs/RPIP-59.md [hunk 7]
```diff
@@ -31,9 +31,9 @@ This RPIP is part of a set of proposals motivated by a desire to rework Rocket P
 ### Deposit queue specification
 ETH from the deposit pool SHALL be matched with validator deposits from queues as follows:
 - There SHALL be a `standard_queue`
-  - When adding a validator, users MAY place their deposit on the `standard_queue`  
+  - When adding a validator, users SHALL be able to place their deposit on the `standard_queue`
```
Comment (LongForWisdom) [inline RPIPs/RPIP-59.md:34]:
```suggestion
  - When adding a validator, users SHALL be able to place their deposit in the `standard_queue`  
```

Comment (LongForWisdom) [reply RPIPs/RPIP-59.md:34]: Minor note. On the queue? Is this common wording? In the UK I would say 'in the queue'. Clearly if there is any authority here on the proper queuing terminology, it belongs on this side of the atlantic. 

Yes. This comment is mostly here as a joke, and you can remove without actioning.

File: RPIPs/RPIP-59.md [hunk 8]
```diff
@@ -31,9 +31,9 @@ This RPIP is part of a set of proposals motivated by a desire to rework Rocket P
 ### Deposit queue specification
 ETH from the deposit pool SHALL be matched with validator deposits from queues as follows:
 - There SHALL be a `standard_queue`
-  - When adding a validator, users MAY place their deposit on the `standard_queue`  
+  - When adding a validator, users SHALL be able to place their deposit on the `standard_queue`  
 - There SHALL be an `express_queue`
-  - When adding a validator, users MAY place their deposit on the `express_queue` by spending one `express_queue_ticket`
+  - When adding a validator, users SHALL be able to place their deposit on the `express_queue` by spending one `express_queue_ticket`
```
Comment (LongForWisdom) [inline RPIPs/RPIP-59.md:36]:
```suggestion
  - When adding a validator, users SHALL be able to place their deposit in the `express_queue` by spending one `express_queue_ticket`
```

File: RPIPs/RPIP-59.md [hunk 9]
```diff
@@ -31,9 +31,9 @@ This RPIP is part of a set of proposals motivated by a desire to rework Rocket P
...
 - When matching ETH from the deposit pool to queued deposits:
   - First, ETH SHALL be matched to the oldest deposit in the `express_queue`; this is repeated `express_queue_rate` times
```
Comment (LongForWisdom) [inline RPIPs/RPIP-59.md:38]: Ahah! We use _in_ here.

**Review by Valdorff — APPROVED** [review]
LGTM

**Comment by orangesamus** [comment]
:thumbsup:

**Pull request merged**